Adyen/Core/Core Protocols/PaymentComponent.swift

```diff
@@ -14,10 +14,6 @@ public protocol PaymentMethodAware: AnyObject {
 
 /// A component that handles the initial phase of getting payment details to initiate a payment.
 public protocol PaymentComponent: Component, PartialPaymentOrderAware, PaymentMethodAware {
-
-    /// The context object for this component.
-    @_spi(AdyenInternal)
-    var context: AdyenContext { get }
```

Comment:
why is this removed? any `PaymentComponent` must have a `AdyenContext ` object, right?